.travis.yml

```diff
@@ -0,0 +1,51 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+dist: trusty
+
+language: java
+
+jdk:
+  - oraclejdk8
+
+addons:
+  apt:
+    sources:
+    - r-packages-trusty
+    packages:
+    - r-base-dev
+
+cache:
+  apt: true
+  directories:
+# caching .m2 causes an error loading hadoop-yarn-common-2.6.0.jar. Not sure why.
+#    - ${HOME}/.m2
+    - ${HOME}/R
+    - /usr/local/lib/R/site-library
+
+install:
+  - sudo Rscript ./src/test/scripts/installDependencies.R
+
+before_script:
+  - chmod -R 755 *
```

Comment:
Are you making all files executable?

Reply:
remove since we added a fix in the code.